win/WIN_ButtonUtils.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include <SDL.h>
```

Comment:
Probably rename this WIN_SDLUtils.h (and the cpp).
**COULD**

Reply:
done